docs/testing.md

```diff
@@ -0,0 +1,68 @@
+# Testing
+
+We want to maintain a project with a high coverage (aiming for 100%). Our main [coverage criterion](https://en.wikipedia.org/wiki/Code_coverage#Coverage_criteria) is **branch coverage**, that is making sure all possible code branches are covered by test. [Coveralls](https://coveralls.io/github/mozilla/addons-frontend) helps us monitor the status of our test suite coverage and it is a good idea to double-check its output when you are opening a Pull Request. You can also [generate a coverage report locally](https://github.com/mozilla/addons-frontend/#code-coverage).
```

Comment:
The second-to-last sentence here could be a bit more concise and probably cut down. It might flow better simply as:

```markdown
[Coveralls](https://coveralls.io/github/mozilla/addons-frontend) monitors our test coverage for every pull request. You can make sure you haven't decreased coverage by checking the Pull Request on GitHub.
```

The tests will actually fail if test coverage drops (enough).

Reply:
Sounds good to me